docs/source/examples/simpleflight.py

```diff
@@ -46,24 +45,24 @@
 else:
     W_w_strc = 8.71e-5*(N_ult*A**1.5*(W_0*W*S)**0.5)/tau
     W_w_surf = 45.24 * S
-equations += [W_w >= W_w_surf + W_w_strc]
+constraints += [W_w >= W_w_surf + W_w_strc]
 
-equations += [D >= 0.5*rho*S*C_D*V**2,
-              Re <= (rho/mu)*V*(S/A)**0.5,
-              C_f >= 0.074/Re**0.2,
-              W <= 0.5*rho*S*C_L*V**2,
-              W <= 0.5*rho*S*C_Lmax*V_min**2,
-              W >= W_0 + W_w]
+constraints += [D >= 0.5*rho*S*C_D*V**2,
+                Re <= (rho/mu)*V*(S/A)**0.5,
+                C_f >= 0.074/Re**0.2,
+                W <= 0.5*rho*S*C_L*V**2,
+                W <= 0.5*rho*S*C_Lmax*V_min**2,
+                W >= W_0 + W_w]
 
 
 def m():
-    return gpkit.Model(D, equations)
+    return gpkit.Model(D, constraints)
```

Comment:
You are referring to the function name right? Making the change, but I just realised it doesn't get called anywhere. Do we still want to leave it in?